### PR TITLE
Super2017 Prices

### PR DESCRIPTION
**Pull request by earl7399**
Fixes https://github.com/magfest/magprime/issues/93

### CODE REVIEW

File: event-prime.yaml [hunk 1]
```diff
@@ -129,14 +129,17 @@ uber::config::badge_prices:
...
-    - { '2015-12-01': 65 }
-    - { '2016-01-01': 70 }
-    - { '2016-02-01': 75 }
-    - { '2016-02-17': 80 }
-
+    - { '2015-09-01': 55 }
+    - { '2015-10-01': 60 }
+    - { '2015-11-01': 65 }
+    - { '2015-12-01': 70 }
+    - { '2016-01-01': 80 }
```
Comment (EliAndrewC) [inline event-prime.yaml]: I think you meant to say ``2016`` and ``2017``.  These dates will NOT do what we want.

Comment (earl7399) [inline event-prime.yaml]: innocent whistling...

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086

**Comment by EliAndrewC** [comment]
+1

**Comment by kitsuta** [comment]
@nickthenewbie1 Just want to double-check that this is what you actually wanted - Bob and I discussed the wording of your list but weren't super sure that this is the correct interpretation. According to the code, the price next to each date / num of badges is what the price will _turn into_ once that limit is reached.

**Comment by earl7399** [comment]
Pinging @nickthenewbie1 ...

**Comment by nickthenewbie1** [comment]
Read your config entries, and the interpretation in this pull request is indeed correct. The issue lists max point at which a price will be accepted.

**Comment by binary1230** [comment]
+1 THOUGH, only note is that, this could be split into two config options with something like this:

(made up dates/numbers/prices)
```
attendee_prices_by_date:
- { '2016-10-31', 55 }
- { '2016-11-31', 65 }

attendee_prices_by_tickets_sold:
- { '4000', 55 }
- { '5000', 65 }
```

the advantage there is that the code in uber doesn't have to guess if '4000' is a date or if it's a number of badges.

if you already have the code differentiating correctly between dates and badges sold, then don't worry about it. if not, it might be easier to split this up.  could also probably do something like:

```
attendee:
  tier:
    price: 55
    ticket_less_than: 4000
    deadline: '2016-11-01'
  tier:
    price: 65
    ticket_less_than: 5000
    deadline: '2016-12-01'
```

**Pull request merged**